Adds the Acera Small Mercenary Ship

kyres1 opened the pull request:
## About the PR
Adds a low-mid-tier mercenary/bounty hunting ship, the Acera. Comes with a brig cell, EMP weapons, a rotary and an autocannon. Comes with good crew facilities, an exped computer and a shield gen. It's really flimsy, all interior walls are also the downgraded version of shuttle walls and it's only enterable via EVA which further adds to its quirkiness. At least it looks pretty.

Comes with boltable doors and lightswitches because why not

## Why / Balance
I was asked to make another merc ship, and also the current ones are pretty one-tone, ugly or tedious to use. I also wanted to add something that made sense to bounty hunt in

## How to test
load it

## Media
<img width="528" height="47" alt="Content Client_qEUKaTwS3m" src="https://github.com/user-attachments/assets/80a162c2-6be0-4f9e-9d1e-f6831b4922b7" />

<img width="1108" height="854" alt="Content Client_DxjIjsM32m" src="https://github.com/user-attachments/assets/3c1ca8d4-a166-4184-918b-a933725d4f56" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ X ] I have added media to this PR or it does not require an ingame showcase.
- [ X ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
hope not

**Changelog**
:cl:
- add: Added the Acera bounty hunting ship

kyres1 committed:
aceraaaaa

kyres1 committed:
adds gravgen and gyroscope and fixes the power

kyres1 committed:
appends desc with an imporatnt detail

monolith8319 committed:
chore: automatically update REUSE headers

thoughtlessuser commented:
have you made sure that those walls have even similar HP to reinforced walls considering power creep

Also if its for combat id suggest just giving it plastitanium for at least a single layer. Mercs are allowed some pretty beefy modifications from recent updates.

kyres1 commented:
> have you made sure that those walls have even similar HP to reinforced walls considering power creep
> 
> Also if its for combat id suggest just giving it plastitanium for at least a single layer. Mercs are allowed some pretty beefy modifications from recent updates.

Nah it was meant to be flimsy, even if the walls have hp it's still got lots of window coverage and the internals use the non-reinforced wall variant

OneZerooo0 commented:
CL?

thoughtlessuser commented:
<img width="245" height="80" alt="Screenshot from 2025-10-16 18-58-12" src="https://github.com/user-attachments/assets/ea143402-83b9-4609-ac9a-ad99c81e5281" />
<img width="265" height="41" alt="Screenshot from 2025-10-16 18-57-38" src="https://github.com/user-attachments/assets/1112dc0d-0cf3-496a-a2f6-14f475653e9d" />

You want to add the BecomeStation component with the shuttles ID (lowercases only) somewhere inside of UID: 1
You also want to change the word "grid" to the shuttles name (also only lowercase) inside the MetaData component near the UID: 1 line

Both of these are taken place inside of the ships serialized YAML file (the grid you saved)

MAKE SURE that the dashes [ - ] are all in line with the other dashes and anything you type isnt out of place. Dont forget to delete any extra spacing you might make between the Type:'s

Danchi299 commented:
```
Arbitrage possible on Acera. Minimal price should be 79159.57973966227, 0% over the appraise price (79159.57973966227).
Assert.That(vessel.Price, Is.AtLeast(idealMinPrice))
  Expected: greater than or equal to 79159.579739662266d
  But was:  63500
```

kyres1 committed:
adjusts price to 83500

kyres1 committed:
Merge remote-tracking branch 'kyres1-origin/10-16-2025-ship_addition_acera' into 10-16-2025-ship_addition_acera

kyres1 commented:
> ```
> Arbitrage possible on Acera. Minimal price should be 79159.57973966227, 0% over the appraise price (79159.57973966227).
> Assert.That(vessel.Price, Is.AtLeast(idealMinPrice))
>   Expected: greater than or equal to 79159.579739662266d
>   But was:  63500
> ```

adjusted to 83500



> CL?

Changed it to look like everyone else's but the current format when making a PR doesnt seem to work for changelogs at all



> <img alt="Screenshot from 2025-10-16 18-58-12" width="245" height="80" src="https://private-user-images.githubusercontent.com/212026398/502326801-ea143402-83b9-4609-ac9a-ad99c81e5281.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEwMjI4MDUsIm5iZiI6MTc2MTAyMjUwNSwicGF0aCI6Ii8yMTIwMjYzOTgvNTAyMzI2ODAxLWVhMTQzNDAyLTgzYjktNDYwOS1hYzlhLWFkOTljODFlNTI4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyMVQwNDU1MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MjhlYjlmNjJhMTc2YjM1Mzc3OWIzZTg3YWQwOWJkZmI1N2MxMjM3ZmE4OGE5ZWUwM2ZmYWM5MzQ4N2I2NjRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GhR_Fg53XM-XN0rH9cXQvqo08CULGrozJQR9v3unCB8"> <img alt="Screenshot from 2025-10-16 18-57-38" width="265" height="41" src="https://private-user-images.githubusercontent.com/212026398/502326807-1112dc0d-0cf3-496a-a2f6-14f475653e9d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEwMjI4MDUsIm5iZiI6MTc2MTAyMjUwNSwicGF0aCI6Ii8yMTIwMjYzOTgvNTAyMzI2ODA3LTExMTJkYzBkLTBjZjMtNDk2YS1hMmY2LTE0ZjQ3NTY1M2U5ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyMVQwNDU1MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMTRjZGM1NjA1NWJlOGI2MjVjMzQ1ZGNmZDQwOGEwZWEzN2FhMDM2MmIzZDc5NzdhMTJiNWYwN2JkYjBhOTQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Q2k5e8Hid1Pxc5u6UngFu8LZwnPtjKHKEvsJe9g3d3A">
> 
> You want to add the BecomeStation component with the shuttles ID (lowercases only) somewhere inside of UID: 1 You also want to change the word "grid" to the shuttles name (also only lowercase) inside the MetaData component near the UID: 1 line
> 
> Both of these are taken place inside of the ships serialized YAML file (the grid you saved)
> 
> MAKE SURE that the dashes [ - ] are all in line with the other dashes and anything you type isnt out of place. Dont forget to delete any extra spacing you might make between the Type:'s

Okay we spoke about this briefly on discord but basically all of this works fine ingame and changing it to your proposed changes breaks everything. no idea how to work around that, let me know if there's something im missing here

kyres1 commented:
no clue what the failed check is about, if someone lets me know how to fix it I can fix it though
